commands/unequip.js

```diff
@@ -0,0 +1,129 @@
+const funcall = require("../modules/funcall.js");
+//simple ping command to check if the bot is online.
+const strifecall = require("../modules/strifecall.js");
+exports.type = "sylladex";
+exports.desc = "Unequip a weapon or other equipment"
+exports.use = `";unequip" removes your currently-equipped weapon from your specibus and puts it in your sylladex.
+";unequip [number]" unequips the indicated weapon from your specibus.
+";unequip trinket" unequips your equipped trinket instead.
+";unequip armor" unequips your equipped armor instead.`
+exports.run = (client, message, args) => {
+	var userid = message.guild.id.concat(message.author.id);
+	var charid = client.userMap.get(userid,"possess");
+
+	let sdex = client.charcall.charData(client,charid,"sdex");
+	let scards = client.charcall.charData(client,charid,"scards");
+	let spec = client.charcall.charData(client,charid,"spec");
+	let kinds = client.charcall.charData(client,charid,"kinds");
+
+	var trinket = client.charcall.charData(client,charid,"trinket");
+	var armor = client.charcall.charData(client,charid,"armor");
+	
+	let curWeapon = client.charcall.charData(client,charid,"equip");
+
+	if(!args[0])
+	{
+		//make sure strife specibus is allocated
+		if(kinds.length == 0){
+			message.channel.send(`Your STRIFE SPECIBUS is not currently allocated to a weaponkind, you must ${client.auth.prefix}allocate it before you can ${client.auth.prefix}equip a weapon!`);
+			return;
+		};
+		
+		args[0] = "" + curWeapon;
```

Comment:
We also need to check if there is an item in that first slot, because they can have a specibus allocated but empty.